sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkOperation.scala

```diff
@@ -35,7 +37,7 @@ private[hive] trait SparkOperation extends Operation with Logging {
 
   protected def sqlContext: SQLContext
 
-  protected var statementId = getHandle().getHandleIdentifier().getPublicId().toString()
+  protected val statementId: String = getHandle.getHandleIdentifier.toString
```

Comment:
Can we revert this change? I see places that use `statementId = UUID.randomUUID().toString`. It's better to have a separated PR to unify this part.

Reply:
HandleIdentifer.publicId is actually  UUID.randomUUID() too

Reply:
I will revert this change in this pr